transforms/ember-object/__testfixtures__/object-literal-with-decorators.output.js

```diff
@@ -0,0 +1,160 @@
+import classic from 'ember-classic-decorator';
+import { alias } from '@ember/object/computed';
+import EmberObject, { action, set, computed } from '@ember/object';
+import { dependentKeyCompat } from '@ember/object/compat';
+import { tracked } from '@glimmer/tracking';
+import { attribute, className } from '@ember-decorators/component';
+import { observes, on } from '@ember-decorators/object';
+
+@classic
+class Foo extends EmberObject {
+  // @ember/object
+
+  @action
+  toggleShowing() {
+    set(this, 'isShowing', !this.isShowing);
+  }
+
+  @computed('firstName', 'lastName')
+  fullName() {
+    return `${this.firstName} ${this.lastName}`;
+  }
+
+  // @ember/object/compat
+
+  @dependentKeyCompat
+  fullName2() {
+    return `${this.firstName} ${this.lastName}`;
+  }
+
+  // @ember/object/computed
+
+  @alias('foo')
+  hasAlias = undefined;
```

Comment:
Correct. We do not need to write `= undefined`, and it is not idiomatic. I had some questions in our Telegram chat last week about if/how we can reliably detect if the initial value is necessary before stripping it off. My thought was to always strip `= undefined` but leave it if the user sets a different initial value. With a lot of these decorators, though, the initial value would immediately be overwritten, so that feels a bit strange also. I don't know that we can reliably detect that, unfortunately, so perhaps we could just log a warning telling the user to review the initial value when it happens.